models/prescription.py

```
@@ -69,6 +69,7 @@ class PrescriptionAudit(db.Model):
 
 
 class Patient(db.Model):
+    __table_args__ = {"schema": "demo"}
```

Ruff (0.15.11)

[warning] Mutable default value for class attribute

(RUF012)